web/src/features/panels/zone/ZoneDetails.tsx

```diff
@@ -190,3 +193,30 @@ function ZoneDetailsContent({
 
   return children as JSX.Element;
 }
+
+const useScrollHashIntoView = (isLoading: boolean) => {
+  const { hash: anchor, pathname, search } = useLocation();
+  const navigate = useNavigate();
+  useEffect(() => {
+    if (isLoading) {
+      return;
+    }
+    const chartIds = Object.values<string>(Charts);
+    const anchorId = anchor.slice(1); // remove leading #
```

Comment:
I'd also ensure all of these are lowerCase here by calling to lower case. It will make it a bit more graceful if someone manually messes up the link.